.github/workflows/performance_tests.yml

```diff
@@ -0,0 +1,52 @@
+name: Performance Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron:  '0 9 * * 0'
```

Comment:
- `0`: minute
- `9`: hour
- `*`: day of the month
- `•`: month
- `0`: day of the week (Sunday)